core/integration/module_config_test.go

```diff
@@ -843,7 +843,7 @@ var vcsTestCases = []vcsTestCase{
 	{
 		name:                     "GitHub public",
 		gitTestRepoRef:           "github.com/dagger/dagger-test-modules",
-		gitTestRepoCommit:        "323d56c9ece3492d13f58b8b603d31a7c511cd41",
+		gitTestRepoCommit:        "f1b295cc1bce8eeea33cc3f42f89452c6fb3429e",
```

Comment:
:eyes: the commit IDs for all these module no longer match. Is that intentional? (any chance we could keep them all aligned, so it's clear at a glance that they're all the same content)

Reply:
I tried to do that by just cp'ing the same files in each repo and using the same commit msg, but I'm guessing that's not sufficient to get the exact same hash. Admittedly I didn't even google it yet so I'll do that 😄

Reply:
Yeah, sadly, the commit date will be different in each of those, so you'll get different commit hashes :cry:

Reply:
Gonna merge now before more rebasing needed, will follow up with this

Reply:
Fix here https://github.com/dagger/dagger/pull/9444